app/waf/src/index.ts

```diff
@@ -8,6 +8,7 @@ const prefix = '/adcaas/v1';
 export async function main(options: ApplicationConfig = {}) {
   const app = new WafApplication(options);
   await app.boot();
+  app.keepAliveAdminToken();
```

Comment:
Is it a background job? Or only run once?

Reply:
A background job. You can review the implementation of keepAliveAdminToken, it will keep the auth token active by loopy logging in again before it expires.